src/handlers.ts

```diff
@@ -125,3 +139,62 @@ export async function handleChromiumCheck(): Promise<void> {
     throw new Error(`One or more upgrade checks failed; see the logs for details`);
   }
 }
+
+export async function handleNodeCheck(): Promise<void> {
+  let thisIsFine = true;
+  const d = debug('roller/node:handleNodeCheck()');
+  const github = await getOctokit();
+
+  d('fetching nodejs/node releases');
+  const { data: releases } = await github.repos.listReleases({
+    owner: REPOS.NODE.OWNER,
+    repo: REPOS.NODE.NAME,
+  });
+  const releaseTags = releases.map((r) => r.tag_name);
+
+  d('fetching electron/electron branches');
+  const { data: branches } = await github.repos.listBranches({
+    owner: REPOS.ELECTRON.OWNER,
+    repo: REPOS.ELECTRON.NAME,
+    protected: true,
+  });
+
+  // TODO: Implement node roller rules for release branches
+  const masterBranch = branches.find((branch) => branch.name === 'master');
+
+  d(`getting DEPS for branch ${masterBranch} in electron/electron`);
+  const depsData = await github.repos.getContents({
+    owner:  REPOS.ELECTRON.OWNER,
+    repo: REPOS.ELECTRON.NAME,
+    path: 'DEPS',
+    ref: masterBranch.commit.sha,
```

Comment:
This could just be a ref to `master`, we don't need to resolve master to a SHA?

Reply:
Sounds right. I copied it over from the original code, but I'll change both instances over, since it seems to work that way.